xml/schema/Core/Address.xml

```diff
@@ -181,6 +183,7 @@
     <title>Street Unit</title>
     <type>varchar</type>
     <length>16</length>
+    <import>true</import>
```

Comment:
No, this is a separate issue for RC as specified in the commit message. Will need to be merged to RC4.6.4 for deploying.